docs/pages/guides/using-sentry.mdx

```diff
@@ -333,6 +338,143 @@ Sentry.init({
 
 </Collapsible>
 
+
+
+</Tab>
+
+<Tab label="SDK 50 beta">
+
+## Install and configure Sentry
+
+<Step label="1">
+
+### Sign up for a Sentry account and create a project
+
+Before proceeding with installing Sentry, you'll need to make sure you have created a Sentry account and project. Here's how to do that:
+
+<Step label="1.1">
+
+[Sign up for Sentry](https://sentry.io/signup/) (it has a free tier), and create a project in your
+Dashboard. Take note of your **organization slug**, **project name**, and **`DSN`** as you'll need
+them later:
+
+- **organization slug** is available in your **Organization settings** tab
+- **project name** is available in your project's **Settings** > **Projects** tab (find it in the list)
+- **`DSN`** is available in your project's **Settings** > **Projects** > **Project name** > **Client Keys
+  (DSN)** tab.
+
+</Step>
+
+<Step label="1.2">
+
+Go to the [Sentry API section](https://sentry.io/settings/account/api/auth-tokens/), and create an
+**auth token**. The token requires the scopes: `org:read`, `project:releases`, and
+`project:write`. Save them.
+
+</Step>
+
+Once you have each of these: organization slug, project name, DSN, and auth token, you're all set!
+
+</Step>
+
+<Step label="2">
+
+### Installation
+
+In your project directory, run:
+
+<Terminal cmd={['$ npx expo install @sentry/react-native']} />
+
+</Step>
+
+<Step label="3">
+
+### App configuration
+
+Configuring `@sentry/react-native` can be done through the config plugin in your [app config](/workflow/configuration/).  Add the following to your project's **app.json** or **app.config.js** file:
+
+{/* prettier-ignore */}
+```json app.json
+{
+  "expo": {
+    "plugins": ["sentry-expo", {
+      "organization": "sentry org slug, or use the `SENTRY_ORG` environment variable",
+      "project": "sentry project name, or use the `SENTRY_PROJECT` environment variable"
+    }]
+    /* @hide ... your existing configuration */ /* @end */
+  }
+}
+```
+
+Next, in any environment where you would like to be able to create releases and upload sourcemaps to Sentry, you will need to set the `SENTRY_AUTH_TOKEN` environment variable to your [Sentry auth token](https://docs.sentry.io/product/cli/configuration/). [Learn how to create a token](https://sentry.io/settings/account/api/).
+
+If you are using EAS Build, you can set the `SENTRY_AUTH_TOKEN` environment variable by [creating a secret of the same name](/build-reference/variables/#using-secrets-in-environment-variables).
+
+> **warning** Never commit secrets to your repository. Keep the Sentry auth token in a safe place, outside your repository.
+
+<ConfigReactNative>
+
+If you do not use [Continuous Native Generation (CNG)](https://docs.expo.dev/workflow/continuous-native-generation/), then you should use the [@sentry/wizard](https://docs.sentry.io/platforms/react-native/#install).
```

Comment:
```suggestion
If you do not use [Continuous Native Generation (CNG)](https://docs.expo.dev/workflow/continuous-native-generation/), then you should use the [`@sentry/wizard`](https://docs.sentry.io/platforms/react-native/#install).
```

Since this is a library, let's use inline syntax highlighting here.